app/renderer/components/preferences/syncTab.js

```diff
@@ -502,16 +496,11 @@ class SyncTab extends ImmutableComponent {
 
         <div className={css(styles.settingsListContainerMargin__bottom)}>
           <span className='settingsListTitle' data-l10n-id='syncTitleMessage' />
-          <a href='https://github.com/brave/sync/wiki/Design' rel='noopener' target='_blank'>
-            <span className={cx({
-              fa: true,
-              'fa-question-circle': true
-            })} />
-          </a>
           <div className={cx({
-            settingsListTitle: true,
-            [css(styles.subText)]: true
-          })} data-l10n-id='syncBetaMessage' />
+            settingsListTitle: true
+          })}>
+            Please download the <a className={css(commonStyles.linkText)} href='https://brave.com/download'>new version of Brave for desktop</a>
```

Comment:
needs period at the end

Reply:
also it is slightly better UX to have this open in a new tab instead of the current tab. `target=_blank` has some security pitfalls so you can use `aboutActions.createTabRequested`